src/Command/CommandContext.php

```diff
@@ -61,23 +67,59 @@ public function iRunWithArrayParameters($name, $params)
     }
 
     /**
+     * @param string $name
+     * @param array  $params
+     * @param string $dialogString
+     */
+    public function iRunDialogConsoleWithArrayParameters($name, array $params, $dialogString = 'Y')
+    {
+        $application = $this->initApplication();
+
+        $this->tester = $applicationTester = new ApplicationTester($application);
+        $applicationTester->run(array());
```

Comment:
Yes
For some reason CLI services are not loaded into `Application` otherwise. So `$command = $application->find($name);` would no more work.